### PR TITLE
Fluid-Build: Fix webpack dependency

### PR DESCRIPTION
**Pull request by DLehenbauer**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -61,7 +61,7 @@
     "test:mocha:report": "lerna run test:mocha --stream --no-bail --no-sort -- -- --timeout 4s --reporter xunit --reporter-option output=nyc/mocha-junit-report.xml",
     "test:realsvc": "lerna run test:realsvc --stream --no-bail --no-sort",
     "test:realsvc:report": "lerna run test:realsvc:report --stream --no-bail --no-sort",
-    "test:report": "npm run test:mocha:report && npm run test:jest:report #&& npm run test:realsvc:report",
+    "test:report": "npm run test:mocha:report && npm run test:jest:report && (true || npm run test:realsvc:report)",
```
Comment (curtisman) [inline package.json:64]: not sure this work on windows either

File: tools/build-tools/src/fluidBuild/tasks/leaf/webpackTask.ts [hunk 2]
```diff
@@ -65,6 +65,11 @@ export class WebpackTask extends LeafWithDoneFileTask {
                 if (this.addChildTask(dependentTasks, child, "npm run build:copy")) {
                     this.logVerboseDependency(child, "build:copy");
                 }
+            } else if (this.addChildTask(dependentTasks, child, "npm run build")) {
```
Comment (curtisman) [inline tools/build-tools/src/fluidBuild/tasks/leaf/webpackTask.ts:68]: `npm run webpack` might be the better dependency?

File: tools/build-tools/src/fluidBuild/tasks/leaf/webpackTask.ts [hunk 3]
```diff
@@ -65,6 +65,11 @@ export class WebpackTask extends LeafWithDoneFileTask {
...
+                this.logVerboseDependency(child, "build");
+                if (this.addChildTask(dependentTasks, child, "npm run build:copy")) {
+                    this.logVerboseDependency(child, "build:copy");
+                }
```
[the remark this answers is not in the document]
Comment (curtisman) [reply tools/build-tools/src/fluidBuild/tasks/leaf/webpackTask.ts:72]: `npm run webpack` might be better?

File: package.json (hunk 1 above)
Comment (DLehenbauer) [reply package.json]: Good grief...  Sorry about that!  Can't go wrong w/echo...

**Comment by DLehenbauer** [comment]
@karlbom - FYI - I think this fixes the build error you mentioned this morning.

The dependency is from 'webpack -> webpack' instead of 'tsc -> webpack' because downstream 'tsc' only consumes the *.d.ts files of upstream packages.

**Review by tylerbutler — APPROVED** [review]
:shipit:

**Comment by DLehenbauer** [comment]
Thanks @tylerbutler !

**Pull request merged**